### PR TITLE
Add configuration for CDB oracle databases

### PR DESCRIPTION
**Pull request by prees1**

### CODE REVIEW

File: pages/tidal tools/analyze_database.md [hunk 1]
```diff
@@ -61,6 +61,8 @@ For example, in Oracle databases, the Data Dictionary and AWR repository tables
 
 You can can create a user with the [script and set of permissions defined in this script](/oracle_user.sql). You should provide a secure password on the first line.
 
+If you are using a CDB database you will also need to create a second user to access to CDB, you can use [this script](/oracle_cdb_user.sql) and also provide a secure password at the top.
```
Comment (pshevtsov) [inline pages/tidal tools/analyze_database.md:64]: “Click here” links will work awful in PDF

**Review by jmariomejiap — APPROVED** [review]
Nice improvement, thanks @prees1

File: pages/tidal tools/analyze_database.md (hunk 1 above)
Comment (prees1) [reply pages/tidal tools/analyze_database.md:64]: Yea agreed. How can we handle this better?

Given the size and length of the script file, its too large to put on the page directly. My thought is, we will have to adjust this content specifically for the PDF output vs the website.

Comment (pshevtsov) [reply pages/tidal tools/analyze_database.md:64]: https://github.com/tidalmigrations/tidalmigrations.github.io/blob/master/oracle_cdb_user.sql is just a seven lines long, I believe it's OK to embed it directly. WDYT?

File: pages/tidal tools/analyze_database.md [hunk 2]
```diff
@@ -61,6 +61,18 @@ For example, in Oracle databases, the Data Dictionary and AWR repository tables
 
 You can can create a user with the [script and set of permissions defined in this script](/oracle_user.sql). You should provide a secure password on the first line.
 
+If you are using a CDB database you will also need to create a second user to access the CDB. You can do that with this script and also provide a secure password at the top:
+
+```
```
Comment (pshevtsov) [inline pages/tidal tools/analyze_database.md:66]: Adding `sql` will add code highlighting:

```sql
CREATE USER c##tidal_comm_user IDENTIFIED BY "replace_this_with_secure_password" account unlock;
GRANT CREATE SESSION to c##tidal_comm_user;
GRANT SELECT ON gv_$archive_dest to c##tidal_comm_user;
GRANT SELECT ON gv_$instance to c##tidal_comm_user;
GRANT SELECT ON v_$managed_standby to c##tidal_comm_user;
GRANT SELECT ON v_$database to c##tidal_comm_user;
GRANT SELECT ON dba_hist_sysmetric_summary to c##tidal_comm_user;
```

**Pull request merged**